From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <heftig@archlinux.org>
Date: Sun, 4 Sep 2022 00:16:08 +0000
Subject: [PATCH] Add missing cdata update in genmarshal tests

Otherwise the test is flaky, as it may try to include a header that's
not generated yet.
---
 test cases/frameworks/7 gnome/genmarshal/meson.build | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/test cases/frameworks/7 gnome/genmarshal/meson.build b/test cases/frameworks/7 gnome/genmarshal/meson.build
index 7686b4bbeab1..9a2cd7a0007d 100644
--- a/test cases/frameworks/7 gnome/genmarshal/meson.build	
+++ b/test cases/frameworks/7 gnome/genmarshal/meson.build	
@@ -39,6 +39,9 @@ foreach mlist : mlists
   marshaller_c = marshallers[0]
   marshaller_h = marshallers[1]
 
+  cdata = configuration_data()
+  cdata.set_quoted('MARSHALLER_HEADER', 'marshaller-@0@.h'.format(idx))
+
   main_c = configure_file(input: 'main.c.in',
     output: 'main-@0@.c'.format(idx),
     configuration: cdata)
